lib/languages.ts

```diff
@@ -783,6 +783,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    wasm: {
+        name: 'WASM',
+        monaco: 'wat',
```

Comment:
I've named the monaco mode 'wat' since that is technically the name of textual format for webassembly, but I'm not sure If I'm just making things more complicated than they need to be.

Reply:
I think this is ok, very much appreciate that you took the time to create the monaco mode!